### PR TITLE
cn: more tests to handler_msg_test.go

### PR DESCRIPTION
**Pull request by ehnuje**
## Proposed changes

- `TestHandleReceiptMsg` and `TestHandleNodeDataMsg` are added.
- `MockProtocolManagerDownloader` is added to mock `ProtocolManagerDownloader` interface.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by ehnuje** [comment]
@KimKyungup @nogang @jeonghonah Dear reviewers, could you please take a look into this?

File: node/cn/handler_msg_test.go [hunk 1]
```diff
@@ -29,6 +30,8 @@ import (
 	"testing"
 )
 
+var err = errors.New("some error")
```
Comment (KimKyungup) [inline node/cn/handler_msg_test.go]: This `err` is used?

Comment (ehnuje) [reply node/cn/handler_msg_test.go]: @KimKyungup Yes, line 252 and 288.

File: node/cn/handler_test.go [hunk 2]
```diff
@@ -101,6 +101,13 @@ func newBlock(blockNum int) *types.Block {
...
 
+func newReceipt(gasUsed int) *types.Receipt {
+	rct := types.NewReceipt(uint(gasUsed), common.Hash{}, uint64(gasUsed))
+	rct.Logs = []*types.Log{}
+	rct.Bloom = types.Bloom{}
+	return rct
+}
+
 func TestNewProtocolManager(t *testing.T) {
 	//1. If consensus.Engine returns an empty Protocol, NewProtocolManager throws an error.
```
Comment (jeonghonah) [inline node/cn/handler_test.go]:
```suggestion
	// If consensus.Engine returns an empty Protocol, NewProtocolManager throws an error.
```

**Pull request merged**